resources/js/Pages/Galleries/MyGalleries/Components/CreateGalleryButton.tsx

```diff
@@ -12,18 +12,20 @@ export const CreateGalleryButton = ({
     className,
     isDisabled,
     onClick,
+    disableIfNoNfts = false,
 }: {
-    nftCount: number;
+    nftCount?: number;
     variant?: ButtonVariant;
     className?: string;
     isDisabled?: boolean;
     onClick?: () => void;
+    disableIfNoNfts?: boolean;
 }): JSX.Element => {
     const { t } = useTranslation();
 
     const createGalleryUrl = route("my-galleries.create");
 
-    if (nftCount === 0) {
+    if (disableIfNoNfts && nftCount === 0) {
```

Comment:
why not make this nftCount optional and check if it's set instead? If not set, then don't disable. If set, disable when it's 0